opm/simulators/wells/MultisegmentWell_impl.hpp

```diff
@@ -2758,8 +2772,8 @@ namespace Opm
             vol_ratio += mix[comp_idx] / b[comp_idx];
         }
 
-        // segment volume
-        const double volume = segmentSet()[seg_idx].volume();
+        // We increase the segment volume with a factor 10 to stabilize the system.
+        const double volume = segmentSet()[seg_idx].volume() * 10;
```

Comment:
Hi, different from the standard well, the volume for MSW has physical meaning. I am not sure we should change the equation here.

Reply:
Would it be an idea start volume factor which is adjusted to 1 at convergence?

Reply:
This function is only used for the accumulation term. It really helps signification on the convergence.

Reply:
> Would it be an idea start volume factor which is adjusted to 1 at convergence?

If we see any negative effect on the results with this that could be good idea.

Reply:
As @hnil  suggests, you might be able to use this as a intermediate term for numerical reason. But the final result should converge with the correct equation, otherwise it might be dangerous for future. When the transition behavoir within the wellbore matters for some cases, this can lead to wrong result.

Reply:
I agree that the factor 10 seems very hacky but I can not image the physical meaning of the accumulation term to be significant even for MSW. We usually take timesteps in the order of days and the volume of a well is very small. I have atleast not been able to find a case where this factor changes anything significant on the results. A motivation for this "hack" is to avoid cases where you get epsilon influx in the last segment of an injector. With this hack the well has a bit more room for accumulating this extra fluid instead of pushing it upwards. 

@GitPaean Do you have a case where you might suspect this term to have any effect on the results? 

To facilitate further testing I will make the hard-coded number run-time parameters.

Reply:
This makes me nervous, too. I like the suggestion to start off larger, then tighten it, but I don't see how we should approach it: "if this is the last iteration, use the real volume" fails since we never know if it is the last iteration...